core/tests/protractor_utils/LibraryPage.js

```diff
@@ -164,7 +166,7 @@ var LibraryPage = function() {
     var explorationCard = await allExplorationsTitled(explorationName).first();
     await waitFor.visibilityOf(
       explorationCard, 'Unable to find exploration ' + explorationName);
-    await explorationCard.click();
+    await action.click('exploration card', explorationCard);
```

Comment:
```suggestion
    await action.click(explorationName + ' exploration card', explorationCard);
```

This will help us locate the point of failure easily.